config/plugins/interactive_environments/cellxgene/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:bionic
```

Comment:
You could base this container ontop of `python:3.6.8-stretch` and avoid installing python3 and all the other dependencies. Only the pip install would be left.